contracts/controller/UController.sol

```diff
@@ -98,14 +94,14 @@ contract UController is ControllerInterface {
         require(nativeToken.owner() == address(this));
         require(nativeReputation.owner() == address(this));
         //To guaranty uniqueness for the reputation systems.
-        require(!reputations[address(nativeReputation)]);
+        require(!actors[address(nativeReputation)]);
+        actors[address(nativeReputation)] = true;
         //To guaranty uniqueness for the nativeToken.
-        require(!tokens[address(nativeToken)]);
+        require(!actors[address(nativeToken)]);
+        actors[address(nativeToken)] = true;
```

Comment:
Should we not do the same for the avatar?

Reply:
see e5c68db